### ISSUE TITLE
Confirm action for Reply all

### ISSUE THREAD

**Issue by severus-tux** [k9mail/k-9#5101]
Can we please have a confirmation dialogue for the "reply all" button? I have been in really cringy positions because of this. I'm sure many would have. Some dialogue which says  "This is a mail from the Director. Everyone in the entire university is in the Cc. Are you out of your mind?"

**Comment by AakaFosfor** [comment]
If, than please with an option "don't show anymore".

**Comment by chaserene** [comment]
agreed. to generalize the idea, a confirmation on what to do whenever there's multiple recipients in the original would be useful. my problem is the inverse of OP's: I often reply only the sender when in fact everyone should receive my reply.

**Comment by ni720** [comment]
Maybe an option in the settings to switch from a reply-all to a reply button would be useful.
